.storybook/webpack.config.js

```diff
@@ -11,7 +11,7 @@ module.exports = ({ config }) => {
 
   config.module.rules.push({
     test: /\.scss$/,
-    use: ['style-loader', 'css-loader', 'sass-loader'],
+    use: ['style-loader', 'css-loader?modules=true', 'sass-loader'],
```

Comment:
This enables `cssModules` in storybook. Currently it doesn't work (and any storybook changes are very easy to miss).